From: Roger Shimizu <rogershimizu@gmail.com>
Date: Mon, 18 Sep 2017 23:34:10 +0900
Subject: Fix scripts/build_deb.sh for debian jessie

Thanks to Tom <hexuxin@foxmail.com> for the bug report.

(cherry picked from commit da85fa17d4b6526e4ffceb6da0465734a252963f)
---
 scripts/build_deb.sh | 4 +++-
 1 file changed, 3 insertions(+), 1 deletion(-)

diff --git a/scripts/build_deb.sh b/scripts/build_deb.sh
index c38f915..28cda54 100755
--- a/scripts/build_deb.sh
+++ b/scripts/build_deb.sh
@@ -65,6 +65,8 @@ apt_init() {
 			sudo sh -c "printf \"\\ndeb $REPO ${OSVER}-backports-sloppy main\" >> /etc/apt/sources.list.d/${OSVER}-backports.list"
 		sudo apt-get update
 		sudo apt-get install --no-install-recommends -y -t $BPO $DEPS_BPO
+		[ -n "$DEPS_EXTRA" -a "$DEPS_EXTRA" = "sloppy" ] &&
+			sudo apt-get install --no-install-recommends -y -t ${BPO}-sloppy $DEPS_BPO
 	else
 		sudo apt-get update
 	fi
@@ -396,7 +398,7 @@ esac
 
 case "$OSVER" in
 jessie)
-	BPO="debhelper libbloom-dev"
+	BPO="debhelper libbloom-dev libsodium-dev"
 	BPOEXTRA=sloppy
 	;;
 stretch)
